core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -68,37 +69,53 @@ export class ExplorationHtmlFormatterService {
       interactionCustomizationArgs: InteractionCustomizationArgs,
       parentHasLastAnswerProperty: boolean,
       labelForFocusTarget: string,
-      savedSolution: Solution): string {
+      savedSolution: string | undefined | null): string {
```

Comment:
Can you please explain this change? Why are we passing in a string now?

Reply:
Also in what scenario is it undefined vs. null?

Reply:
When this change was first made, not all instances had a fifth argument added in code. So those would be undefined. I went through all of them and added null as a value. The reason for passing a string is now explained in the doc string above the function.